managed-ledger/src/main/java/org/apache/bookkeeper/mledger/impl/ManagedLedgerImpl.java

```diff
@@ -2764,6 +2764,7 @@ private void offloadLoop(CompletableFuture<PositionImpl> promise, Queue<LedgerIn
                                            scheduledExecutor, name)
                             .whenComplete((ignore2, exception) -> {
                                     if (exception != null) {
+                                        log.error("Failed to offload data for ledgerId {}", ledgerId, exception);
```

Comment:
It's better to also print the managed ledger name so that we can distinguish which topic occurs the problem.

Reply:
Ok, I'll fix.

Reply:
For consistency, it might be helpful to also add the ledger's name to the warning log that occurs when the cleanup offload fails (line 2962).